server/cmd/bblfsh-web/main.go

```diff
@@ -33,8 +41,14 @@ func main() {
 		return
 	}
 
+	if level, ok := logLevels[os.Getenv("LOG_LEVEL")]; ok {
```

Comment:
What if it's not set?

Reply:
Then `ok` will be false (also if it's passed an unknown log level), and no level will be explicitly used, so it will use logrus default (as it's currently doing)

Reply:
oh right, 🤦‍♂. thanks!